python/graphstorm/config/argument.py

```diff
@@ -386,6 +393,21 @@ def freeze_lm_encoder_epochs(self):
 
         return 0
 
+    @property
+    def glem(self):
+        """ Setting up (default) GLEM configs 
+        """
+        if hasattr(self, "_glem"):
```

Comment:
i don't understand how it is set up. users only need to turn on the flag?

Reply:
Yes. Please see the example config file: [training_scripts/gsgnn_np/ml_nc_utext_glem.yml](https://github.com/awslabs/graphstorm/pull/262/files/7ef2d3751671094e48aeff17e7130dbbb5f2c9ae#diff-c86b86ea495b966bb658770fdefb4414d74bea3242e0718167fc4b5de397b036).